docs/modules/ROOT/pages/vshn-managed/nextcloud/create.adoc

```diff
@@ -0,0 +1,57 @@
+= Create a Nextcloud service
+
+Apply the following object on your namespace, as specified by its YAML description.
+
+.Example to create a Nextcloud instance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNNextcloud
+metadata:
+  name: nextcloud-app1-prod # <1>
+  namespace: prod-app # <2>
+spec:
+  parameters:
+    service:
+      version: "23" # <3>
+      useExternalPostgreSQL: true # <4>
+      postgreSQLParameters: {} # <5>
+    size: # <6>
+      plan: standard-2
+  writeConnectionSecretToRef:
+    name: nextcloud-creds # <7>
+----
+<1> Instance name
+<2> The namespace where the object will be created
+<3> Nextcloud version
+<4> If we should use the VSHNPostgreSQL as a database backend instead of the build-in SqLite database backend. (defaults to true)
+<5> PostgreSQL database parameters. See xref:vshn-managed/postgresql/index.adoc[PostgreSQL by VSHN] (Only used when `useExternalPostgreSQL` is false)
+<6> Size of the Nextcloud instance. See xref:vshn-managed/nextcloud/plans.adoc[Plans and Sizing] for more information.
+<7> Secret where the connection details are provisioned. This secret shouldn't exist before creation.
```

Comment:
```suggestion
<7> Secret where the connection details are provisioned. This secret is created after the instance is provisioned.
```

Reply:
No. The secret shouldn't exist before the creation or crossplane will not write anything as an "unmanaged" sercret already exists.

Reply:
Ah now I see from what point of view you are referring to. Fine by me.